install/_test_setup.sh

```diff
@@ -21,7 +25,7 @@ setup() {
       D)
         rm "$_SANDBOX/$filepath"
         ;;
-      A | M | AM)
+      A | M | AM | ??)
```

Comment:
No idea what this is, comments? :D

Reply:
Done in cde39996b19350443764541682b14102b3e84086. :)